core-api/src/main/java/com/sequenceiq/cloudbreak/api/service/ExposedServiceCollector.java

```diff
@@ -128,12 +132,20 @@ public ExposedService getByName(String name) {
 
     public Set<String> getFullServiceListBasedOnList(Collection<String> services, Optional<String> bpVersion) {
         Set<String> result = new HashSet<>(services);
-        if (services.contains("ALL")) {
+        if (services.contains(ALL)) {
             result = getAllKnoxExposed(bpVersion);
         }
         return result;
     }
 
+    public Set<String> getFullServiceListByNameBasedOnList(Collection<String> services, Optional<String> bpVersion) {
```

Comment:
Could you please rename it?
`getFullServiceListWithDefaultValues` would be more understandable here.
Don't we need to remove the `ALL` service type after adding all Knox exposed service name the services as default contains it?